source/install/packstack.html.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Packstack quickstart
+title: Packstack
 category: documentation
 authors: apevec, dneary, garrett, jasonbrooks, jlibosva, mmagr, pixelbeat, pmyers,
   rbowen, gbraad, cbrown2
 wiki_category: Documentation
-wiki_title: Quickstart
+wiki_title: Packstack
```

Comment:
@apevec happy to abandon this one and just do one patch that removes these fields?